Dockerfile

```
@@ -9,4 +9,6 @@ FROM debian:buster-slim
 WORKDIR /app
 #RUN apt-get update && apt-get install -y extra-runtime-dependencies && rm -rf /var/lib/apt/lists/*
 COPY --from=builder /xd_bot /app/xd_bot
+VOLUME ./config.toml /app/config.toml
```

This should be (per the spec at https://docs.docker.com/engine/reference/builder/#volume)
```suggestion
VOLUME /app/config.toml
```